src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -163,7 +159,7 @@ public RuntimeImpl(Transaction tx, BlockCapsule block, DepositImpl deposit,
   }
 
 
-  public void precompiled() throws ContractValidateException, ContractExeException {
+  private void precompiled() throws ContractValidateException, ContractExeException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the redundant '!unknownSymbol!' thrown exception declaration(s). ([squid:RedundantThrowsDeclarationCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ARedundantThrowsDeclarationCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnEHMySgN4uCcljsrs&open=AWbnEHMySgN4uCcljsrs)

[ISSUE_KEY]: AWbnEHMySgN4uCcljsrs